install/deb/proftpd/tls.conf

```diff
@@ -0,0 +1,62 @@
+#
+# Proftpd sample configuration for FTPS connections.
+#
+# Note that FTPS impose some limitations in NAT traversing.
+# See http://www.castaglia.org/proftpd/doc/contrib/ProFTPD-mini-HOWTO-TLS.html
+# for more information.
+#
+<IfModule mod_dso.c>
+    # If mod_tls was built as a shared/DSO module, load it
+    LoadModule mod_tls.c
+</IfModule>
+<IfModule mod_tls.c>
+TLSEngine                               on
+TLSLog                                  /var/log/proftpd/tls.log
+# this is an example of protocols, proftp works witl all, but use only the most secure ones like TLSv1.1 and TLSv1.2
+TLSProtocol                             TLSv1.1 TLSv1.2
```

Comment:
ubuntu 16.04.7:
if included the `TLSv1.3` proftpd fails to start with error: 
`unknown protocol: 'TLSv1.3' on line 15 of '/etc/proftpd/tls.conf'`

Reply:
You have point 

Ubuntu 20.04 
root@test:/etc/proftpd# proftpd -v
ProFTPD Version 1.3.6c